configs/nextflow.hopper.config

```diff
@@ -302,4 +302,48 @@ myeloid {
     params.gens_accessdir = "/access/myeloid_const/plot_data"
     params.accessdir = "/access/myeloid_const/"
   } 
+
+modycf {
+    params.outdir = "${params.resultsdir}${params.dev_suffix}"
+    params.subdir = 'modycf'
+    params.crondir = "${params.outdir}/cron/"
+    params.intersect_bed = "/fs1/resources/ref/hg38/bed/twist-modycf/MODY_CF_updated.bed"
```

Comment:
A suggestion here, what about starting to write protect these files as a "soft" protection from accidental removal? I.e. `chmod -w MODY_CF_updated.bed`

Reply:
good idea.